### DISCUSSION TITLE
how can i make lazyVim transparent ?

### DISCUSSION THREAD

**Discussion by BekaAbate** [LazyVim/LazyVim#1991] [Q&A]
I can't find a way to make lazyVim transparent , i tried enabling transparency in tokyoNight but  Neo tree and telescope are still not transparent..

**Comment by dpetka2001** [comment]
See this [comment](https://github.com/LazyVim/LazyVim/discussions/116#discussioncomment-4763452). I tried this and both neo-tree and telescope are transparent.
